sensor/common/scan/scan.go

```diff
@@ -270,10 +348,17 @@ func (s *LocalScan) fetchSignatures(ctx context.Context, errorList *errorhelpers
 	// Fetch signatures from cluster-local registry.
 	sigs, err := s.fetchSignaturesWithRetry(ctx, signatures.NewSignatureFetcher(), image, image.GetName().GetFullName(), registry)
 	if err != nil {
-		log.Debugf("Failed fetching signatures for image %q: %v", image.GetName().GetFullName(), err)
+		// Like Central, only log errors related to fetching signatures.
+		if !errors.Is(err, errox.NotAuthorized) {
+			log.Errorf("Fetching image signatures for image %q: %v", image.GetName().GetFullName(), err)
+		} else {
+			// Log not authorized errors in debug mode, since we expect them to occur.
```

Comment:
why is this expected?

Reply:
That comment was copied from Central's enricher [here](https://github.com/stackrox/stackrox/blob/919d8f0c9b3c04c53ab5e436044e34c854d9e666/pkg/images/enricher/enricher_impl.go#L742-L748).

I do not know the history, but based on the testing I was doing that led to this change I can say some registries will report a `401` if you try to access something that doesn't exist. In the case of signatures, ACS is trying to pull a `sha256...sig` file from the registry, which will only exists if a signature has been created - which is uncommon.